### DISCUSSION TITLE
Upgrade to latest on outlinewiki/outline shows old version on installation sidebar UI

### DISCUSSION THREAD

**Discussion by decentral1se** [outline/outline#3202] [Self Hosting]
After upgrading to `0.62.0` from `0.60.1`, it still shows the older version in the sidebar.

But we're running `outlinewiki/outline:0.62.0` and have run `yarn db:migrate --env=production-ssl-disabled`:

```
/opt/outline $ yarn db:migrate --env=production-ssl-disabled
yarn run v1.22.17
$ sequelize db:migrate --env=production-ssl-disabled

Sequelize CLI [Node: 16.14.0, CLI: 6.3.0, ORM: 6.9.0]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".
No migrations were executed, database schema was already up to date.
Done in 1.08s.
/opt/outline $ 
```

This is a self-hosted deployment using the official docker image. Here are our configs for the deployment:

> https://git.coopcloud.tech/coop-cloud/outline/src/branch/main/compose.yml

**To Reproduce**
1. Upgrade to latest with self-hosted docker deployment

**Expected behavior**
The new version is shown in the UI.

**Screenshots**

![image](https://user-images.githubusercontent.com/1991377/156725936-6a712366-79e1-456c-b510-56408b4fc89f.png)

**Outline (please complete the following information):**
- Install: self hosted
- Version: https://hub.docker.com/layers/outlinewiki/outline/0.62.0/images/sha256-9b53b86f9488618e3d5ed0017ee1ff460d48063bd6d81270807bd9cf026580e1?context=explore

**Desktop (please complete the following information):**
 - OS: Debian
 - Browser: Chrome
 - Version: 98.0.4758.102

**Mobile (please complete the following information):**
 - Device: [e.g. iPhone6] N/A
 - OS: [e.g. iOS8.1] N/A
 - Browser [e.g. stock browser, safari] N/A

**Comment by decentral1se** [comment]
I **think** this may be related to mounting a persistent volume in https://git.coopcloud.tech/coop-cloud/outline/src/commit/0ad7813b7be32d1fad2397a61588a85e64ace848/compose.yml#L21 but then that raises the question, is there a path I should be mounting under `/opt/outline` as a persistent volume for keeping data?  If I remove this volume, things do break. I don't see this hinted at in https://github.com/outline/outline/blob/main/Dockerfile or https://github.com/outline/outline/blob/main/docker-compose.yml.

**Comment by decentral1se** [comment]
```
/opt/outline $ cat package.json | grep -i version
  "version": "0.60.1"
```

Aha!

**Comment by decentral1se** [comment]
Removing this mounted volume for the new version results in this error:

![image](https://user-images.githubusercontent.com/1991377/156729351-0112d0f3-7908-46e7-8286-bd8ffe20ad47.png)

**Reply by tommoor** [reply]
You must always run database migrations when updating, the errors are due to missing columns
